ffi/ecma402/src/pluralrules.rs

```diff
@@ -331,20 +331,25 @@ mod testing {
             },
         ];
         let provider = icu_testdata::get_provider();
-        for test in tests {
-            let l: Locale = test.locale.parse().expect("locale exists");
-            let l = crate::Locale::FromLocale(l);
-            let plr = super::PluralRules::try_new_with_provider(l, test.clone().opts, &provider)?;
-            let actual = test
-                .numbers
-                .iter()
-                .map(|n| {
-                    let mut result = String::new();
-                    plr.select(*n, &mut result).unwrap();
-                    result
-                })
-                .collect::<Vec<String>>();
-            assert_eq!(test.expected, actual, "for test case: {:?}", &test);
+        for (i, test) in tests.into_iter().enumerate() {
```

Comment:
The formatting makes this look like a bigger change than it is. This could just be simplified because we can now write `Locale`-typed expressions concisely.